### DISCUSSION TITLE
Contacting a GitHub programmer/collaborator...

### DISCUSSION THREAD

**Discussion by rmonroe041** [community/community#56060] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I contact a GitHub programmer/collaborator? I need said programmer to update one of his/her posted files.

**Comment by airtower-luna** [comment, marked as answer]
Check the documentation (README, etc.) of the repository in question for how to report problems. If those don't say anything, creating an issue in the repository is usually good.
